### DISCUSSION TITLE
Middle click to minimize?

### DISCUSSION THREAD

**Discussion by xr280xr** [ramensoftware/windhawk-mods#752] [Mod ideas]
Windhawk has been a productivity life-saver when involuntarily moving from my 7+ Taskbar Tweaker-enabled Windows 10 machine.  I always liked the middle click a taskbar item to minimize the window option. Any chance of adding an alternate minimize instead of close option to the "Middle click to close on the taskbar" mod?

**Comment by m417z** [comment]
Thanks for the suggestion. It's planned. You can also vote for it here under **Middle click: Switch to / Minimize / Close**:
https://ramensoftware.com/windhawk-mods-for-the-windows-11-taskbar
